packages/lib-react-components/src/CloseButton/CloseButton.spec.js

```diff
@@ -9,12 +9,6 @@ describe('Component > CloseButton', function () {
     expect(screen.getByRole('button').hasAttribute('disabled')).to.be.false()
   })
 
-  it('renders the light button', function () {
-    const LightStory = composeStory(Light, Meta)
-    render(<LightStory />)
-    expect(screen.getByRole('button').hasAttribute('disabled')).to.be.false()
-  })
```

Comment:
Removed this test because it does not test anything different from the Default Story.